components/sr-slot-spacing/component.html

```diff
@@ -0,0 +1,56 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html" />
+
+<polymer-element name="sr-slot-spacing">
+    <template>
+        <style>
+            h2 {
+                margin: 1em 0 0;
+                color: inherit;
+                font-size: 0.6em;
+            }
+        </style>
+
+        <h2>
+            {{ slotDuration | formatDuration }} slots,
+            {{ gameDuration | formatDuration }} games,
+            {{ gapDuration | formatDuration }} gap
+        </h2>
```

Comment:
This is more of an informational element than it is a heading, ideally this would be a `<p>` I think.